client/state/reducer.js

```diff
@@ -37,7 +37,6 @@ import pageTemplates from './page-templates/reducer';
 import postFormats from './post-formats/reducer';
 import receipts from './receipts/reducer';
 import rewind from './rewind/reducer';
-import selectedEditor from './selected-editor/reducer';
```

Comment:
I'm getting eslint errors because of a rule that was added here [to prevent new legacy reducers from being added](https://github.com/Automattic/wp-calypso/pull/41878#issue-414220016).